components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/authz/OAuth2AuthzEndpoint.java

```diff
@@ -2404,6 +2445,33 @@ private String doUserAuthorization(OAuthMessage oAuthMessage, String sessionData
         }
     }
 
+    /**
+     * Validate scopes before consent page
+     *
```

Comment:
Shall we add a full stope at the end of each comment line. Use the same approach we used in other places